changelog/unreleased/features/1703--flexible-plugin-loading.md

```diff
@@ -0,0 +1,9 @@
+The options `vast.plugins` and `vast.plugin-dirs` may now be specified on the
+command line as well as the configuration. Use the options `--plugins` and
+`--plugin-dirs` respectively.
+
+The build-time CMake option `VAST_ENABLE_PLUGIN_AUTOLOADING` is superseded by a
```

Comment:
Not sure if we should mention VAST_ENABLE_PLUGIN_AUTOLOADING, since it was never a public-facing option.